samples/BenchmarkDotNet.Samples/BenchmarkDotNet.Samples.csproj

```diff
@@ -19,10 +20,12 @@
   <ItemGroup>
     <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
     <PackageReference Include="System.Drawing.Common" Version="4.7.2" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.7.2" />
```

Comment:
Unfortunately this is required as this library provides and runs the vstest executables which load the test adapter for everything to work. It doesn't work as a transitive dependency. I spent a long time searching and wasn't able to find any way to avoid this. Perhaps if we want to avoid this we could try speaking to the VSTest team to see if they have any suggestions.